plf-packaging-resources/src/main/resources/CHANGE_LOG.txt

```diff
@@ -8,50 +8,59 @@ eXo ECMS ${org.exoplatform.platform.version}
+    
+    
+eXo Integration ${org.exoplatform.platform.version}
 *************************
 
 ** Bug
-    * [JCR-2469] - The indexation fails if a node doesn't have a primary type
+    * [MOB-2197] - Download file Button does not work on document application
+    
+    
+eXo Wiki ${org.exoplatform.platform.version}
+*************************
+
+** Improvement
+    * [WIKI-1400] - When a page is made public, it becomes editable by anyone
 	
 eXo Social ${org.exoplatform.platform.version}
 *************************
 
 ** Bug
-    * [SOC-6175] - Loading Icon is still displayed when click on Load More button
-    * [SOC-6260] - The filter alphabetically in space members tab doesn't work
-    
-    
-** Improvement    
-    * [SOC-6261] - Space Members - Make the identities sort field extensible
+    * [SOC-6249] - Keep receiving mention notification everytime someone comment the activity 
+    * [SOC-6270] - Cannot preview documents in activites stream
 
     
 eXo Calendar ${org.exoplatform.platform.version}
 *************************
 
 ** Bug
-    * [CAL-1464] - Ignore selected calendars user preference in space calendar app
+    * [CAL-1480] - [Event Edit] Find a time popup does not contain event participants and does not respect Time Format
+    * [CAL-1491] - Date-picker does not show the selected value after initial selection
     
+    
+eXo Chat 2.2.4
+**************
+
+** Bug
+    * [CHAT-1015] - Page is reloaded when the websocket handshake fails   
 
 eXo Platform ${org.exoplatform.platform.version}
 *************************
 
 ** Bug
-    * [PLF-7990] - [Oracle DB] empty page when displaying more than 1000 users in people page
-    * [PLF-8220] - Creating Task projects: Primary_key violation error
-    * [PLF-8320] - JCR Session leak detected when launching the exo server
-    * [PLF-8346] - Document App: deleting default folder Symlink doesn't work 
-    * [PLF-8347] - Document App: default folder Symlink cut/paste doesn't work
-    * [PLF-8406] - [MOB] Sharing a photo from mobile app breaks the activity stream
-    * [PLF-8408] - Unified search similar types' value issue
-    * [PLF-8414] - Unable to edit a user without email
-    * [PLF-8420] - Login History migration: Node not found when cleaning JCR 's Nodes
+    * [MOB-2248] - The design of "Enregistrer" button in event creation
+    * [MOB-2258] - Display problem in spaces makes it difficult to access space applications and Status button
+    * [PLF-8348] - Documents App: Wrong "warning" content displayed
+    * [PLF-8462] - Allow users to classify uploaded files in Activity stream
     
 
 ** Improvement
-    * [PLF-8380] - [MOB] Can't comment directly from activity stream on mobile
-    * [PLF-8411] - [MOB] Use new images preview when sharing it from mobile app
+    * [PLF-8459] - Allow users to classify uploaded files in Activity stream
+    * [PLF-8379] - Disabled users are in the members list of the space
```

Comment:
We may have to revert for 5.2.4 this or backport CHAT-1026 and fix SOC-6267 (which is more difficult)